.github/workflows/build.yml

```diff
@@ -46,16 +46,28 @@ jobs:
     name: Build Web
     needs: build
     runs-on: ubuntu-latest
-    strategy:
-        matrix:
-            node: [ 14, 16 ]
```

Comment:
I removed required checks in repo settings for web 14 and 16. What will be default node version in this case?

Reply:
> I removed required checks in repo settings for web 14 and 16. What will be default node version in this case? 

16

Reply:
Some increases are justified so it's should be up to the team to decide